match2/wikis/rainbow_six/match_summary.lua

```diff
@@ -0,0 +1,591 @@
+local Class = require('Module:Class')
+local DisplayHelper = require('Module:MatchGroup/Display/Helper')
+local Json = require('Module:Json')
+local Logic = require('Module:Logic')
+local MatchGroupUtil = require('Module:MatchGroup/Util')
+local MatchSummary = require('Module:MatchSummary/Base')
+local OperatorIcon = require('Module:OperatorIcon')
+local OpponentDisplay = require('Module:OpponentDisplay')
+local Table = require('Module:Table')
+local Template = require('Module:Template')
+
+local _GREEN_CHECK = '[[File:GreenCheck.png|14x14px|link=]]'
+local _NO_CHECK = '[[File:NoCheck.png|link=]]'
+local _ROUND_ICONS = {
+	atk = '[[File:R6S Para Bellum atk logo.png|14px|link=]]',
+	def = '[[File:R6S Para Bellum def logo.png|14px|link=]]',
+	otatk = '[[File:R6S Para Bellum atk logo ot rounds.png|11px|link=]]',
+	otdef = '[[File:R6S Para Bellum def logo ot rounds.png|11px|link=]]',
+}
+local _ARROW_LEFT = '[[File:Arrow sans left.svg|15x15px|link=|Left team starts]]'
+local _ARROW_RIGHT = '[[File:Arrow sans right.svg|15x15px|link=|Right team starts]]'
+local _LINK_DATA = {
+	vod = {icon = 'File:VOD Icon.png', text = 'Watch VOD'},
+	preview = {icon = 'File:Preview Icon.png', text = 'Preview'},
+	lrthread = {icon = 'File:LiveReport.png', text = 'LiveReport.png'},
+	siegegg = {icon = 'File:SiegeGG icon.png', text = 'SiegeGG Match Page'},
+	opl = {icon = 'File:OPL Icon.png', text = 'OPL Match Page'},
+	esl = {icon = 'File:ESL icon.png', text = 'Match page on ESL'},
+	faceit = {icon = 'File:FACEIT-icon.png', text = 'Match page on FACEIT'},
+	lpl = {icon = 'File:LPL Play icon.png', text = 'Match page on LPL Play'},
+	r6esports = {icon = 'File:Copa Elite Six icon.png', text = 'R6 Esports LATAM Match Page'},
+	stats = {icon = 'File:Match_Info_Stats.png', text = 'Match Statistics'},
+}
+
+-- Operator Bans Class
+
+local OperatorBans = Class.new(
+	function(self)
+		self.root = mw.html.create('table')
+		self.text = ''
+	end
+)
+
+function OperatorBans:setLeft()
+	self.root
+		:addClass('brkts-popup-body-operator-bans')
+		:css('float', 'left')
+
+	return self
+end
+
+function OperatorBans:setRight()
+	self.root
+		:addClass('brkts-popup-body-operator-bans')
+		:css('float', 'right')
+
+	return self
+end
+
+function OperatorBans:add(operator)
+	if Logic.isEmpty(operator) then
+		return self
+	end
+	self.root
+		:tag('tr')
+			:tag('td')
+				:css('padding', '0')
+				:tag('div')
+					:wikitext(OperatorIcon.getImage{operator,'50x50px'})
+	return self
+end
+
+function OperatorBans:create()
+	self.root:wikitext(self.text)
+	return self.root
+end
+
+-- Score Class, both for the "big" score, and the halfs scores
+
+local Score = Class.new(
+	function(self)
+		self.root = mw.html.create('div'):css('width','70px')
+		self.table = self.root:tag('table'):css('line-height', '20px')
+		self.top = mw.html.create('tr')
+		self.bottom = mw.html.create('tr')
+	end
+)
+
+function Score:setLeft()
+	--self.root
+		--:css('text-align', 'left')
+
```

Comment:
This right?

Reply:
I've uncommented it now, but shouldn't be needed unless a language which is right to left is added, as left-align is default. But just in case it is, why not have the support already.